packages/sdks/overrides/angular/src/constants/target.ts

```diff
@@ -0,0 +1,3 @@
+type Target = import('../types/targets.js').Target;
+/** This file should be overriden for each framework. Ideally this would be implemented in Mitosis.  */
+export const TARGET = 'angular' as Target;
```

Comment:
one thing missing here is adding `angular` to this `Target` type:

https://github.com/BuilderIO/builder/blob/30f58449160ec2cd30e94e7c36c710e683c43056/packages/sdks/src/types/targets.ts#L1

Reply:
done!